src/sentry/static/sentry/app/components/search/searchResultWrapper.jsx

```diff
@@ -3,11 +3,17 @@ import {css} from '@emotion/core';
 import React from 'react';
 import omit from 'lodash/omit';
 
-const SearchResultWrapper = styled(props => <div {...omit(props, 'highlighted')} />)`
+const SearchResultWrapper = styled(props => (
+  <div
+    {...omit(props, 'highlighted')}
+    ref={element => props.highlighted && element?.scrollIntoView?.({block: 'nearest'})}
+  />
+))`
   cursor: pointer;
   display: block;
   color: ${p => p.theme.gray800};
   padding: 10px;
+  scroll-margin: 120px;
```

Comment:
I'm not familiar with the` scroll-margin` property, but are you sure that it works well on safari together with the `scrollIntoView` function?

Reply:
yup :)

Reply:
Oh on safari, no I don't know

Reply:
From what I could find [scroll-margin doesnt' work well with scrollIntoView](https://caniuse.com/?search=scroll-margin) with safari.

Reply:
That’s unfortunate, but I think this is still a net improvement